etc/database/remove_analysis.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
```

Comment:
The other scripts in this directory use a '-' character instead of an underscore to separate the words.  Lets update this one to match them.

Reply:
The same for the add-user script that was migrated.

Reply:
The names have been updated and should be reflected in the READMEs they're referenced in.